pkgs/development/compilers/idris2/default.nix

```
@@ -1,22 +1,29 @@
-{ lib, stdenv, fetchFromGitHub, makeWrapper
-, clang, chez
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, clang
+, chez
+, gmp
 }:
 
+# NOTICE: An `idris2WithPackages` is available at: https://github.com/claymager/idris2-pkgs
+
 # Uses scheme to bootstrap the build of idris2
 stdenv.mkDerivation rec {
   pname = "idris2";
-  version = "0.3.0";
+  version = "0.4.0";
 
   src = fetchFromGitHub {
     owner = "idris-lang";
     repo = "Idris2";
     rev = "v${version}";
-    sha256 = "0sa2lpb7n6xqfknwld9rzm4bnb6qcd0ja1n63cnc5v8wdzr8q7kh";
+    sha256 = "105jybjf5s0k6003qzfxchzsfcpsxip180bh3mdmi74d464d0h8g";
   };
 
   strictDeps = true;
   nativeBuildInputs = [ makeWrapper clang chez ];
-  buildInputs = [ chez ];
+  buildInputs = [ gmp ];
```

Can we add something in the commit message that explains why we add `gmp`? Doesn't have to be huge, but if we change something we should leave minimal bread crumbs for the interested reader to figure out what's going on (including me, since I forgot most of what's going on here).

---

It is a build dependency of idris2 as of v0.4.0. I could add that to the commit message.

---

```suggestion
  buildInputs = [ chez gmp ];
```

(since removed from propagatedNativeBuildInputs)

---

```suggestion
  buildInputs = [ chez gmp ];
```

(since removed from propagatedNativeBuildInputs in previous suggestion)